src/wasm/wasm-ir-builder.cpp

```diff
@@ -1404,7 +1406,17 @@ Result<> IRBuilder::makeBreak(Index label, bool isConditional) {
   // Use a dummy condition value if we need to pop a condition.
   curr.condition = isConditional ? &curr : nullptr;
   CHECK_ERR(ChildPopper{*this}.visitBreak(&curr, *labelType));
-  push(builder.makeBreak(curr.name, curr.value, curr.condition));
+  auto* br = builder.makeBreak(curr.name, curr.value, curr.condition);
+  push(br);
+
+  if (likely) {
+    // Branches are only possible inside functions.
+    assert(func);
+    Function::CodeAnnotation annotation;
+    annotation.branchLikely = likely;
+    func->codeAnnotations[br] = annotation;
```

Comment:
We can depend on default initialization to simplify:
```suggestion
    func->codeAnnotations[br].branchLikely = likely;
```